crates/spirv-std/src/lib.rs

```diff
@@ -93,8 +91,9 @@
 //! Core functions, traits, and more that make up a "standard library" for SPIR-V for use in
 //! rust-gpu.
 
-#[cfg_attr(not(target_arch = "spirv"), macro_use)]
+#[macro_use]
 pub extern crate spirv_std_macros as macros;
```

Comment:
You marked my previous comment asking about the `pub` here as resolved, but I don't remember what the explanation was, sorry - it's just that `spirv_std::macros::...` is used elsewhere, right?

Reply:
Oh I did this because the things in `spirv-std` now need `spirv_std::spirv` to be visible for all platforms. This way I didn't need to write `use spirv_std::spirv;` in every single file.